.github/CODEOWNERS

```diff
@@ -25,3 +25,4 @@
 /vertex_ai/             @terraform-google-modules/dee-data-ai @terraform-google-modules/terraform-samples-reviewers
 /vpc/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /workflows/             @terraform-google-modules/torus-dpe @terraform-google-modules/terraform-samples-reviewers
+/managedkafka/          @terraform-google-modules/managedkafka-dev-team @terraform-google-modules/terraform-samples-reviewers
```

Comment:
This line was reporting an unknown error, because while this team does exist, it didn't have permissions to this repo. 

I have manually granted this team permissions, per the samples runbook.

Reply:
Thank you!